zyte_common_items/base.py

```diff
@@ -75,7 +76,7 @@ def from_dict(cls, item: Optional[Dict]):
     @classmethod
     def _from_dict(cls, item: Optional[Dict], *, trail: _Trail = None):
         """Read an item from a dictionary."""
-        if not item:
+        if item is None:
```

Comment:
So that `{}` turns into an empty instance of the corresponding type, instead of `None`.